client-registry-api/src/main/java/uk/gov/di/authentication/clientregistry/entity/ClientRegistrationRequest.java

```diff
@@ -150,6 +157,7 @@ public ClientRegistrationRequest(
             clientType = ClientType.WEB.getValue();
         }
         this.clientType = clientType;
+        this.idTokenSigningAlgorithm = idTokenSigningAlgorithm;
```

Comment:
We need to make this default to es256 so that this is not a breaking API change. As-is I think this would stop the self service tool from working until they updated to send the new field?